dklearn/cross_validation.py

```diff
@@ -398,5 +412,5 @@ def train_test_split(*arrays, **options):
     if options:
         raise ValueError("Invalid parameters passed: %s" % str(options))
 
-    seed = different_seeds(1, random_state=random_state)[0]
+    seed, = random_state_data(1, random_state=random_state)
```

Comment:
It's interesting that people find it both more and less readable. I've run into enough accidental trailing comma issues that I'm slightly wary of any, even if intentional :).